Dockerfile

```diff
@@ -22,10 +24,15 @@ RUN pip install --upgrade pip \
 RUN curl -L https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash; \
     helm init --client-only
 
-# Install kubectl
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl; \
-    mv kubectl /usr/bin/kubectl; \
-    chmod +x /usr/bin/kubectl
+# Install cfssl and cfssljson
+RUN curl -sSL https://pkg.cfssl.org/R1.2/cfssl_linux-amd64 > /usr/bin/cfssl \
+ && curl -sSL https://pkg.cfssl.org/R1.2/cfssljson_linux-amd64 > /usr/bin/cfssljson \
+ && chmod +x /usr/bin/cfssl /usr/bin/cfssljson
+
+# Install kubectl and kubeadm
+RUN curl -L https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64/kubectl > /usr/bin/kubectl \
+ && curl -L https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64/kubeadm > /usr/bin/kubeadm \
```

Comment:
and I would like to have the URL possible to be given as an `ARG`. You could default it, if you want 🙂 

Something like
```Dockerfile
ARG KUBE_BINARY_URL="https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64"
RUN curl -L --remote-name ${KUBE_BINARY_URL}/kubectl
```

What do you think about that? Is it too much, or does it make the code more readable/maintainable (that would be my hope)?